### DISCUSSION TITLE
Biset module updates

### DISCUSSION THREAD

**Discussion by topazcoding** [DISTRHO/Cardinal#680] [General]
The biset tracker is cool and thanks 👍 for adding them. 

Is there a chance they will be updated to the latest versions please ?

**Comment by dromer** [comment]
When someone gets around to it.

Is there any specific change that warrants an update? (a major bugfix for instance)

I see that they updated the license from MIT to GPL-3.0-or-later.
Not an issue for us, but quite a radical change.
